test/forgery_test.exs

```diff
@@ -17,7 +17,7 @@ defmodule ForgeryTest do
     end
   end
 
-  doctest Forgery, import: true
+  # doctest Forgery, import: true
```

Comment:
accident?

Reply:
No, in my machine I have this error:
```
== Compilation error in file test/forgery_test.exs ==
** (ExUnit.DocTest.Error) lib/forgery.ex:28: expected non-blank line to follow iex> prompt
```
I need to investigate this further